NorthstarDLL/concommand.h

```diff
@@ -83,8 +83,7 @@ typedef void (*FnCommandCallback_t)(const CCommand& command);
 //-----------------------------------------------------------------------------
 // Returns 0 to COMMAND_COMPLETION_MAXITEMS worth of completion strings
 //-----------------------------------------------------------------------------
-typedef int (*__fastcall FnCommandCompletionCallback)(
-	const char* partial, char commands[COMMAND_COMPLETION_MAXITEMS][COMMAND_COMPLETION_ITEM_LENGTH]);
+typedef int (*FnCommandCompletionCallback)(const char* partial, char commands[COMMAND_COMPLETION_MAXITEMS][COMMAND_COMPLETION_ITEM_LENGTH]);
```

Comment:
Why?

Reply:
There's a warning (in both MSVC and gcc) that `__fastcall` doesn't do anything in a function pointer typedef and because this file is part of the preprocessed header it floods the output, which was making it hard to look for errors.